openstef/postprocessing/postprocessing.py

```diff
@@ -39,16 +46,39 @@ def normalize_and_convert_weather_data_for_splitting(weather_data):
         / np.percentile(weather_data["radiation"].dropna(), 99.0)
         * -1
     )
-    wind_ref = weather_features.calculate_windspeed_at_hubheight(
+    wind_ref_series = weather_features.calculate_windspeed_at_hubheight(
         weather_data["windspeed_100m"], fromheight=100
     )
-    wind_ref = wind_ref / np.abs(np.amax(wind_ref)) * -1
+    wind_ref = wind_ref_series.to_frame()
```

Comment:
I see that the old line of code did a normalization AND *-1.
Currently, a unit test fails. This will be solved if you add a `wind_ref *= -1` here again

![image](https://user-images.githubusercontent.com/18208480/179238261-e6a8a9b5-64ba-463f-81aa-c63f480d57e5.png)